### PR TITLE
making loading validation rules allot easier

### PR DESCRIPTION
**Pull request by danfernand**

### CODE REVIEW

File: src/components/form.js [hunk 1]
```diff
@@ -54,6 +54,9 @@ class Form extends React.Component {
     }
     return form.errors || null;
   }
+  loadValidationRules(rules) {
+    this.dispatch(loadValidationRules(this.formID, rules || {}));
```
Comment (dkoston) [inline src/components/form.js]: it's easy to see in this diff as I can see `import {.., loadValidationRules } from '../actions';` above however, this looks like a recursive function call at first glance (second glance helps you see the extra params to the inner call). 

I'd advise to change the function name here for enhanced readability.

Comment (danfernand) [inline src/components/form.js]: i agree, done!

**Comment by danfernand** [comment]
ready for review

**Comment by dkoston** [comment]
> redux-react-form@0.1.4 /Users/dave/hdc-repos/redux-react-form
├── eslint-config-airbnb@5.0.1 
├── UNMET PEER DEPENDENCY eslint-plugin-react@^3.16.1
├── history@2.0.0 
├─┬ lodash.uniqueid@4.0.0 
│ └── lodash.tostring@4.1.1 
└── react-router@2.0.0 

npm WARN eslint-config-airbnb@5.0.1 requires a peer of eslint-plugin-react@^3.16.1 but none was installed.

**Comment by dkoston** [comment]
otherwise, LGTM

**Pull request merged**